### PR TITLE
Layouts 

### PR DESCRIPTION
**Pull request by mattgle**
## Summary
Layouts of technology

## Trello Card
https://trello.com/c/DdWYqZOF/23-content-layout
## Screenshots
captura de pantalla 2018-10-24 a la s 15 03 42

### CODE REVIEW

File: maquetado/styles.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
 body {
   font-family: 'Roboto', sans-serif;
+  margin: 10px;
```
Comment (henryzarza) [inline maquetado/styles.scss]: Delete this you should do this with `padding` property. Check this [article](https://medium.com/wolox-driving-innovation/what-margin-padding-taught-me-after-50-projects-6878b53c903f)

File: maquetado/tech.scss [hunk 2]
```diff
@@ -17,14 +18,23 @@
   font-size: $text-medium;
   margin: 10px 0;
 }
+//ojo con los paddig entre los objetos de adentro.
+.tech-container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  padding: 20px 20px;
```
Comment (henryzarza) [inline maquetado/tech.scss]: You can use the shorthand (`padding: 20px`)

File: maquetado/tech.scss [hunk 3]
```diff
@@ -17,14 +18,23 @@
   font-size: $text-medium;
   margin: 10px 0;
 }
+//ojo con los paddig entre los objetos de adentro.
+.tech-container {
```
Comment (henryzarza) [inline maquetado/tech.scss:22]: I think that is necessary that you assign `max-width` and `width` properties

**Review by henryzarza — CHANGES_REQUESTED** [review]
The design isn't the same of the card

**Review by henryzarza — APPROVED** [review]

File: maquetado/index.html [hunk 4]
```diff
@@ -10,11 +10,68 @@
...
-    <span class="tech-description">React is a declarative component-based JavaScript library for building user interfaces.</span>
+  <div class="tech-container">
+    <div class="tech">
+      <img src="assets/react_icon.svg" alt="react icon" class="tech-icon">
+      <h1 class="tech-title">ReactJS</h1> 
+      <h2 class="tech-release">Release: 2013</h2>
+      <span class="tech-description">React is a declarative component-based JavaScript library for building user interfaces.</span>
+    </div>
+    <div class="tech">
+      <img src="assets/angular_icon.svg" alt="react icon" class="tech-icon">
```
Comment (SKOLZ) [inline maquetado/index.html]: `alt="react icon"` this and all the following tech images are not react icons

File: maquetado/index.html [hunk 5]
```diff
@@ -10,11 +10,68 @@
 </head>
 <body>
   <h1 class="title">Front-End Tools</h1>
```
Comment (SKOLZ) [inline maquetado/index.html]: this should be an h2

File: maquetado/index.html [hunk 6]
```diff
@@ -10,11 +10,68 @@
...
-  <div class="tech">
-    <img src="assets/react_icon.svg" alt="react icon" class="tech-icon">
-    <h1 class="tech-title">ReactJS</h1> 
-    <h2 class="tech-release">Release: 2013</h2>
-    <span class="tech-description">React is a declarative component-based JavaScript library for building user interfaces.</span>
+  <div class="tech-container">
+    <div class="tech">
+      <img src="assets/react_icon.svg" alt="react icon" class="tech-icon">
+      <h1 class="tech-title">ReactJS</h1>
```
Comment (SKOLZ) [inline maquetado/index.html]: this should be an h3

File: maquetado/index.html [hunk 7]
```diff
@@ -10,11 +10,68 @@
...
-  <div class="tech">
-    <img src="assets/react_icon.svg" alt="react icon" class="tech-icon">
-    <h1 class="tech-title">ReactJS</h1> 
-    <h2 class="tech-release">Release: 2013</h2>
-    <span class="tech-description">React is a declarative component-based JavaScript library for building user interfaces.</span>
+  <div class="tech-container">
+    <div class="tech">
+      <img src="assets/react_icon.svg" alt="react icon" class="tech-icon">
+      <h1 class="tech-title">ReactJS</h1> 
+      <h2 class="tech-release">Release: 2013</h2>
```
Comment (SKOLZ) [inline maquetado/index.html]: this should be an h4 or a span

**Review by SKOLZ — CHANGES_REQUESTED** [review]

**Review by SKOLZ — APPROVED** [review]

**Pull request merged**